### ISSUE TITLE
import urllib3 raise ImportError(ImportError: urllib3 v2.0 only supports OpenSSL 1.1.1+)

### ISSUE THREAD

**Issue by BrunoArsioli** [drivendata/cookiecutter-data-science#308]
I follow your guidelines for installation
``` pip install cookiecutter ```

then navigate to the directory where I want to start a new project, but 
``` cookiecutter https://github.com/drivendata/cookiecutter-data-science ```

I get the following error:

``` 
Traceback (most recent call last):
  File "/Users/xxx/Library/Python/3.9/bin/cookiecutter", line 5, in <module>
    from cookiecutter.__main__ import main
  File "/Users/xxx/Library/Python/3.9/lib/python/site-packages/cookiecutter/__main__.py", line 2, in <module>
    from cookiecutter.cli import main
  File "/Users/xxx/Library/Python/3.9/lib/python/site-packages/cookiecutter/cli.py", line 22, in <module>
    from cookiecutter.main import cookiecutter
  File "/Users/xxx/Library/Python/3.9/lib/python/site-packages/cookiecutter/main.py", line 17, in <module>
    from cookiecutter.repository import determine_repo_dir
  File "/Users/xxx/Library/Python/3.9/lib/python/site-packages/cookiecutter/repository.py", line 7, in <module>
    from cookiecutter.zipfile import unzip
  File "/Users/xxx/Library/Python/3.9/lib/python/site-packages/cookiecutter/zipfile.py", line 6, in <module>
    import requests
  File "/Users/xxx/Library/Python/3.9/lib/python/site-packages/requests/__init__.py", line 43, in <module>
    import urllib3
  File "/Users/xxx/Library/Python/3.9/lib/python/site-packages/urllib3/__init__.py", line 38, in <module>
    raise ImportError(
ImportError: urllib3 v2.0 only supports OpenSSL 1.1.1+, currently the 'ssl' module is compiled with LibreSSL 2.8.3. See: https://github.com/urllib3/urllib3/issues/2168 
```

the local machine is macbook M2

**Comment by pjbull** [comment]
This isn't a CCDS bug, so unfortunately there's nothing we can do about it.

If you read the error message, it points you to an `urllib3` issue that covers a number of potential solutions, including:
 - [upgrading your python version](https://github.com/urllib3/urllib3/issues/2168#issuecomment-1535053569)
 - [installing `urllib3` at something less than 2](https://github.com/urllib3/urllib3/issues/2168#issuecomment-1535914514)
 - or, as this SO answer reports, [installing openssl@1.1 with brew](https://stackoverflow.com/a/76187415/1692709)

**Comment by BrunoArsioli** [comment]
Than you @pjbull for your msg and guidance! 

The work around for me, was to create a new virtual environment (in the terminal):

``` python3 -m venv .venv ```

This command created a new virtual environment in my directory named .venv.

After creating the virtual environment, I activate it via: 

``` source .venv/bin/activate ```

Once the virtual environment is activated, I see the (.venv) at the beginning of the terminal prompt. 
(This indicates that the virtual environment is active, and any Python packages that I install from now on, will be installed in this environment, rather than being installed globally). (I am learning here)

Now, I installed the older version of urllib3 (in terminal):

``` pip install urllib3==1.26.7 ```

Now, I was able to install cookiecutter in this isolated environment with the command:

``` pip install cookiecutter ```

And, finally, I was able to run: 

``` cookiecutter https://github.com/drivendata/cookiecutter-data-science ``` 

without any issues.
Thank you for your msg. It helped me with the work around solution.
